conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -38,18 +41,17 @@ echo "$config" > ~/.condarc
 conda clean --lock
 
 conda update --yes --all
-
+conda install --yes conda-build==1.18.2
```

Comment:
I have looked a bit closer and I believe the problem was an issue with an assertion in conda-build. See this [comment]( https://github.com/conda-forge/nn-feedstock/pull/3#issuecomment-201329112 ).

It appears like this PR ( https://github.com/conda/conda-build/pull/855 ) fixes the assertion issue. Assuming that does fix the problem, that change is in the 1.20.1 release.

Though I'm not sure whether we can use it or not yet as we have this issue ( https://github.com/conda/conda-build/issues/889 ) on Mac and conceivably it would be present on Linux too. We believe this is fixed in this PR ( https://github.com/conda/conda-build/pull/892 ), but are still waiting for a release.